src/renderer/components/alert/RemainingFundsWillBeUsedAlert.tsx

```diff
@@ -11,7 +12,7 @@ const useStyles = makeStyles((theme) => ({
 
 export default function RemainingFundsWillBeUsedAlert() {
   const classes = useStyles();
-  const balance = useAppSelector((s) => s.balance.balanceValue);
+  const balance = useAppSelector((s) => s.rpc.state.balance);
 
   if (balance == null || balance <= 0) {
     return <></>;
```

Comment:
**Suggestion:** Consider adding error handling for the case when `balance` is `null`. Currently, the code simply returns an empty fragment when `balance` is `null` or less than or equal to 0. It would be more user-friendly to display a message indicating that the balance could not be retrieved.
```suggestion
  const balance = useAppSelector((s) => s.rpc.state.balance);

  if (balance == null || balance <= 0) {
    return <Alert severity="error">Could not retrieve balance</Alert>;
  }
```